doc/source/timeseries.rst

```diff
@@ -1204,7 +1204,7 @@ As the same as `DatetimeIndex`, the endpoints will be included in the result. Be
 
 Frequency Conversion and Resampling with PeriodIndex
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-The frequency of Periods and PeriodIndex can be converted via the ``asfreq``
+The frequency of `Periods` and `PeriodIndex` can be converted via the ``asfreq``
```

Comment:
ah good catch ... i found a few other places where it should be double backticks instead of single